uber/custom_tags.py

```diff
@@ -414,7 +414,7 @@ def render(self, context):
         charge = self.charge.resolve(context)
         cherrypy.session[payment_id] = charge.to_dict()
 
-        email = ""
```

Comment:
Style change, I though I was going to need to modify this custom tag and changed this while I was in this file.  I ended up not needing to make any changes here though.